tests/idem/structs.rs

```diff
@@ -14,6 +14,18 @@ pub struct Foo {
 
 struct Bar;
 
+struct NewInt<T: Copy>(i32, SomeType /* inline comment */, T);
+
+struct Qux<'a,
+           N: Clone + 'a,
+           E: Clone + 'a,
+           G: Labeller<'a, N, E> + GraphWalk<'a, N, E>,
+           W: Write + Copy>
+    (AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA, // Comment
+     BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB,
+     G,
+     W);
```

Comment:
I don't think we can be consistent with both the expression form and regular structs. I think being closer to other items is more important here (and gives less weird indenting). Struct literals also vary a bit from struct definitions, so I think its OK.